lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -124,6 +124,14 @@ export const upgradeableTooling: Record<string, ToolingDefinition> = {
       versioning: `${regexVersioning.id}:^(?<major>\\d+?)\\.(?<minor>\\d+?)\\.(?<patch>\\d+)\\.(?<build>\\d+)$`,
     },
   },
+  conftest: {
+    asdfPluginUrl: 'https://github.com/looztra/asdf-conftest',
+    config: {
+      datasource: GithubReleasesDatasource.id,
```

Comment:
There are no releases here: https://github.com/looztra/asdf-conftest/releases

Reply:
Isn't the release requirement to satisfy the search for `packageName`? i.e. Cosign below is hosted on Gitlab.

Reply:
No

Reply:
The asdfPluginUrl field just seems like a [readme enhancement](https://github.com/renovatebot/renovate/pull/18924/files#diff-85fdd510661eb2bed15c5ed8012f6459467b84a172b4eff855c29ddacb794562R11) with rendered output visible [here](https://docs.renovatebot.com/modules/manager/asdf/#additional-information).

Reply:
Are you sure @rarkins? tflint is in the same file [here](https://github.com/renovatebot/renovate/blob/dab6fa1a4eb1b9b93ab071cadaee18df9e571ccc/lib/modules/manager/asdf/upgradeable-tooling.ts#L703C21-L703C62), and it's `asdfPluginUrl` value is https://github.com/skyzyx/asdf-tflint.git but it doesn't have any github releases either?

Reply:
Sorry, you're right. The releases will be queried from https://github.com/open-policy-agent/conftest/releases/